packages/swap/src/routes/swap.ts

```diff
@@ -295,6 +295,8 @@ router.get(
       depositChannelAffiliateBrokers: affiliateBrokers,
       depositChannelMaxBoostFeeBps: channel?.maxBoostFeeBps,
       effectiveBoostFeeBps,
+      swapBoostedAt: swap?.swapBoostedAt?.valueOf(),
```

Comment:
maybe `depositBoostedAt` would be more correct?

Reply:
now that i think of it, wouldnt this be the same as `depositReceivedAt`

Reply:
and should we have a `boostSkippedAt` too?

Reply:
> now that i think of it, wouldnt this be the same as `depositReceivedAt`

ahh yeah, now i remember why i didn't include it int he first place 🙈